packit/api.py

```diff
@@ -554,21 +554,25 @@ def create_srpm(
             logger.error(f"Failed when getting Bodhi updates: {exc}")
             return []
 
+    @staticmethod
+    async def status_main(status):
+        """
+        Schedule repository data retrieval calls concurrently.
+        :param status:
```

Comment:
Can you please add a type annotation and something to docstring for the `status`? (We are using `mypy` in the [pre-commit](https://github.com/packit-service/packit/blob/master/.pre-commit-config.yaml#L26). The type-check is also run in the CI.) Thanks!

Reply:
@lachmanfrantisek  Done. Please check the changes in the latest commit.